coderd/debug_test.go

```diff
@@ -232,6 +233,52 @@ func TestDebugHealth(t *testing.T) {
 	})
 }
 
+func TestHealthSettings(t *testing.T) {
```

Comment:
1) Can we add a test case where we un-dismiss?
2) This should be audited, so we want to add an auditor and assert that an audit action took place.

Reply:
Something like this perhaps?

https://github.com/coder/coder/blob/main/coderd/workspaces_test.go#L597-L612

https://github.com/coder/coder/blob/main/coderd/workspaces.go#L313-L321

Reply:
1. Undismiss test case done

Reply:
2. I added support for auditing, but I have no idea if this is sufficient. Keep in mind that "health settings" are stored as site configs, so in this case, the resource (and its ID) is artificial.

Reply:
You can add a mock auditor into the unit tests and assert that the changes were audited as expected.